app/assets/stylesheets/application.scss

```diff
@@ -205,17 +194,33 @@ select{padding: initial;}
   background-image: -webkit-linear-gradient(top, whiteSmoke 0%, #EEE 100%);
   background-image: -ms-linear-gradient(top, whiteSmoke 0%, #EEE 100%);
   background-image: -o-linear-gradient(top, whiteSmoke 0%, #EEE 100%);
-  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#f5f5f5', endColorstr='#eeeeee',GradientType=0 );
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#f5f5f5', endColorstr='#eeeeee', GradientType=0);
```

Comment:
@tbrisker, I'm not actively working on it, would you be willing to take
replace this PR ?

thanks!